### PR TITLE
updated the images path in the blog content

### PR DESCRIPTION
**Pull request by indraraj**
- Updated the images asset path in the blog content.

### CODE REVIEW

File: _posts/2020-09-15-debezium-auto-create-topics.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ rkerner
 
 ++++
 <div class="imageblock centered-image">
-    <img src="/assets/images/new_pipes.jpg" class="responsive-image" alt="Create new topics / pipes">
+    <img src="/assets/assets/images/new_pipes.jpg" class="responsive-image" alt="Create new topics / pipes">
```
Comment (mdrillin) [inline _posts/2020-09-15-debezium-auto-create-topics.adoc:15]: This path looks ok as-is

File: _posts/2020-10-08-debezium-community-stories-with-renato-mefi.adoc [hunk 2]
```diff
@@ -19,7 +19,7 @@ In today's edition it's my pleasure to talk to https://twitter.com/renatomefi[Re
 [.centered-image.responsive-image]
 ====
 ++++
-<img src="/assets/images/renatomefi.jpg" style="max-width:50%;" class="responsive-image">
+<img src="/assets/assets/images/renatomefi.jpg" style="max-width:50%;" class="responsive-image">
```
Comment (mdrillin) [inline _posts/2020-10-08-debezium-community-stories-with-renato-mefi.adoc:22]: The 3 image paths in this file look ok as-is

**Review by mdrillin — CHANGES_REQUESTED** [review]
The path changes indicated can be left as-is

**Comment by indraraj** [comment]
Thanks for pointing, I have updated the pr.

File: _posts/2020-09-15-debezium-auto-create-topics.adoc (hunk 1 above)
Comment (indraraj) [reply _posts/2020-09-15-debezium-auto-create-topics.adoc]: reverted

**Review by mdrillin — APPROVED** [review]
looks good, thanks for updating!

**Pull request merged**